code/modules/mob/living/simple_animal/abnormality/_abnormality.dm

```diff
@@ -127,6 +136,31 @@
 				if(patrol_path.len)
 					patrol_move(patrol_path[patrol_path.len])
 
+/mob/living/simple_animal/hostile/abnormality/attackby(obj/O, mob/user, params)
+	if(istype(O, /obj/item/reagent_containers))
+		if(status_flags & GODMODE)
+			if(world.time > chemCooldownTimer)
+				var/obj/item/reagent_containers/myContainer = O
+				harvestChem(myContainer, user)
+			else
+				to_chat(user, "<span class='notice'>You may need to wait a bit longer.</span>")
+		else
+			to_chat(user, "<span class='notice'>This doesn't seem like the right time.</span>")
+	return ..()
+
+/mob/living/simple_animal/hostile/abnormality/proc/harvestChem(obj/item/reagent_containers/C, mob/user)
+	if(harvestPhrase == "dummy")
+		harvestPhrase = "<span class='notice'>You harvest... something... into [C].</span>"
+	if(harvestPhraseThirdPerson == "dummy")
+		harvestPhraseThirdPerson = "[user] harvests... something... into [C]."
+	to_chat(user, "[harvestPhrase]")
+	user.visible_message("[harvestPhraseThirdPerson]")
```

Comment:
Second proc of visible_message is message sent to caller(user) of proc. Don't have to use to_chat.